api/handlers/search/name.go

```diff
@@ -25,11 +26,17 @@ func (n Name) NameSearchHandler(w http.ResponseWriter, r *http.Request) {
 	firstName := r.URL.Query().Get("first_name")
 	lastName := r.URL.Query().Get("last_name")
 	dob := r.URL.Query().Get("dob")
+	userID := r.URL.Query().Get("user_id")
 	communityID := r.URL.Query().Get("community_id")
 
 	zap.S().Debugf("first_name: %v, last_name: %v, dob: %v, community_id: %v", firstName, lastName, dob, communityID)
 
 	//TODO need to check if user is in community before searching just the name
```

Comment:
Yeah I think I was wrong here, we shouldn't need to care if the user is in a community - just want to check if the civilian is in a community.

I think we add a completely different route for loading civilians on like the `civ-dashboard` page. This route should be strictly for the `name-search` route that police and dispatch run multiple times per second.